engine/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -327,11 +327,14 @@ assumes that the Docker daemon is in the `stopped` state.
       "storage-driver": "devicemapper",
        "storage-opts": [
          "dm.thinpooldev=/dev/mapper/docker-thinpool",
-         "dm.use_deferred_removal=true"
+         "dm.use_deferred_removal=true",
+         "dm.use_deferred_deletion=true"
        ]
     }
     ```
 
+    >**Note**: It is suggested to set both `dm.use_deferred_removal=true` and `dm.use_deferred_deletion=true` to prevent unintentional leaking of mount points.
```

Comment:
Avoid "It is suggested". Instead, try something like "Always set both ....to prevent unintentional leaking of mount points."